receiver/activedirectorydsreceiver/README.md

```diff
@@ -1,8 +1,12 @@
 # Active Directory Domain Services Receiver
 
-The `active_directory_ds` receiver scrapes metric relating to an Active Directory domain controller using the Windows Performance Counters.
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
```

Comment:
This component is not currently in any distributions

Reply:
ok.delete